x/staking/types/delegation.go

```diff
@@ -365,32 +342,6 @@ func NewRedelegationEntryResponse(
 	}
 }
 
-// String implements the Stringer interface for RedelegationResp.
-func (r RedelegationResponse) String() string {
-	out := fmt.Sprintf(`Redelegations between:
-  Delegator:                 %s
-  Source Validator:          %s
-  Destination Validator:     %s
-  Entries:
-`,
-		r.DelegatorAddress, r.ValidatorSrcAddress, r.ValidatorDstAddress,
-	)
-
-	for i, entry := range r.Entries {
-		out += fmt.Sprintf(`    Redelegation Entry #%d:
-      Creation height:           %v
-      Min time to unbond (unix): %v
-      Initial Balance:           %s
-      Shares:                    %s
-      Balance:                   %s
-`,
-			i, entry.CreationHeight, entry.CompletionTime, entry.InitialBalance, entry.SharesDst, entry.Balance,
-		)
-	}
-
-	return strings.TrimRight(out, "\n")
-}
-
```

Comment:
Why are we removing this? We need this anyway

Reply:
we should use yaml for `String()`